src/GRA.Data/Repository/PsBranchSelectionRepository.cs

```diff
@@ -85,7 +89,8 @@ public async Task<PsBranchSelection> GetByCodeAsync(string secretCode)
         {
             var selection = DbSet
                 .AsNoTracking()
-                .Where(_ => _.AgeGroupId == ageGroupId && _.BranchId == branchId);
+                .Where(_ => _.AgeGroupId == ageGroupId && _.BranchId == branchId
```

Comment:
If we are going to break on an `&&` we should break on each `&&`

Reply:
Fixed in 7ea0e75f2f09c9275a89a5a37a81ef42cf2b2fba